doc(README) : Adding how to debug the project (#51)
aerogear/mobile-security-service-operator: README.adoc

IMPORTANT: To create all you need be logged as admin. Use `oc login -u system:admin`.

> 

Author: Small fix typo.

IMPORTANT: The local changes are applied when the command `operator-sdk up local --namespace=mobile-security-service-operator` is executed then it is not a hot deploy and to get the latest changes you need re-run the command.

=== Debugging

> Following the steps to debug the project in some IDEs.

Reviewer: ```suggestion
Follow the below steps to debug the project in some IDEs.
```

Author: 👍

> NOTE: The code need to be compiled/built first.

Reviewer: ```suggestion
NOTE: The code needs to be compiled/built first.
```

Author: 👍

==== IntelliJ IDEA / GoLand

[source,shell]
----
$ export WATCH_NAMESPACE=default
$ cd cmd/manager/
$ dlv debug --headless --listen=:2345 --api-version=2
----

Then, debug the project from the IDE by using the default setup of `Go Remote` option.

==== Visual Code

[source,shell]
----
$ export WATCH_NAMESPACE=default
$ $ dlv --listen=:2345 --headless=true --api-version=2 exec ./build/_output/bin/mobile-security-service-operator-local  --
----

> Then, attach to process using the following Visual Code config.

Reviewer: ```suggestion
Then, debug the project using the following Visual Code launch config.
```

Author: 👍
